test/e2e/specs/editor/various/pattern-overrides.spec.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Pattern Overrides', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.activateTheme( 'emptytheme' ),
+			requestUtils.setGutenbergExperiments( [
+				'gutenberg-pattern-partial-syncing',
+			] ),
```

Comment:
I'm a little out of the loop on the pattern overrides work but do we also need to enable the block bindings experiment here i.e. `gutenberg-custom-fields`?

The current test failures look like a bindings object is expected but is missing.
```suggestion
			requestUtils.setGutenbergExperiments( [
				'gutenberg-pattern-partial-syncing',
				'gutenberg-custom-fields',
			] ),
```

Reply:
The current goal is to allow only setting the `pattern-partial-syncing` experiment, but we'll likely combine these two or remove the partial-syncing experiment as it matures. The failure was likely caused by not rebasing correctly on my end 😅.